src/main/java/seedu/address/logic/commands/FindTagCommand.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.util.StringUtil;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Finds and lists all persons in address book that has a tag that contains the given text. Matching is
+ * case insensitive.
+ */
+public class FindTagCommand extends Command {
+    public static final String COMMAND_WORD = "findtag";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose tags contain the specified"
+        + " text (case-insensitive) and displays them in the list.\n"
+        + "Parameters: KEYWORD [TEXT]...”\n";
```

Comment:
Can add an example here, reference to delete command. Also, I dont think you need [TEXT], just KEYWORD should be enough, for standardisation.

MESSAGE_USAGE = COMMAND_WORD
            + ": Deletes the person identified by the index number used in the displayed person list.\n"
            + "Parameters: INDEX (must be a positive integer)\n"
            + "Example: " + COMMAND_WORD + " 1";

Reply:
Ok, changed. Now reads:
findtag: Finds all persons whose tags contain the specified text (case-insensitive) and displays them in the list.
Parameters: TEXT
Example: findtag friend